.vsts-ci/misc-analysis/markdown.yml

```diff
@@ -44,10 +34,6 @@ jobs:
       curl -o- --progress-bar -L https://yarnpkg.com/install.sh | bash
     displayName: Bootstrap Yarn
 
-  - bash: |
-      yarn config  set global-folder "$(YARN_GLOBAL_CACHE_FOLDER)"
-    displayName: Set Yarn global cache folder
-
```

Comment:
leave this, we are supposed to cache at these locations

Reply:
updated